File: DheerajKumar/problem12.cpp
//12. WAP to take input 2 numbers from the user and print their difference.

#include <iostream>
using namespace std;

int diffrence(int number1, int number2){
    return number1 > number2 ? 
        number1 - number2 : number2 - number1;
}

int main(){
    int number1, number2;

    cout<<"Enter 1st number -> ";
    cin>>number1;
    cout<<"Enter 2nd numebr -> ";
    cin>>number2;

    cout<<"The Difference is -> "<<diffrence(number1, number2);
}

File: LovelyKumari/BasicProgramming/Problem12.java
import java.util.Scanner;

public class Problem12 {
    //WAP to take input 2 numbers from the user and print their difference.
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter the first number: ");
        int num1= sc.nextInt();
        System.out.print("Enter the second number: ");
        int num2=sc.nextInt();

        int difference= num1-num2;
        System.out.println("The difference between " + num1 + " and " + num2 + " is: " + difference);

        
    }

    
}
